data.json

```diff
@@ -5847,12 +5847,9 @@
             ]
         }
     },
-    "grsync.png": {
```

Comment:
Because this is a root icon, you'll also have to switch the root icon name around too (for Circle and Square)